frontend/src/features/public-form/components/FormEndPage/FormEndPageContainer.tsx

```diff
@@ -72,14 +72,18 @@ export const FormEndPageContainer = (): JSX.Element | null => {
       />
     )
   }
+
+  // Feedback is not supported on MRF
+  const isFeedbackHidden =
+    form.responseMode === FormResponseMode.Multirespondent
   return (
     <Box py={{ base: '1.5rem', md: '2.5rem' }} w="100%">
       <FormEndPage
         colorTheme={form.startPage.colorTheme}
         submissionData={submissionData}
         formTitle={form.title}
         endPage={form.endPage}
-        isFeedbackSubmitted={isFeedbackSubmitted}
+        hideFeedbackSection={isFeedbackSubmitted || isFeedbackHidden}
```

Comment:
this prop naming change caused a visual regression on storybook, i think. May need to trouble you to update the `FormEndPage`stories to use this new prop!

_Also separate supernit while I'm relooking this:_
Also now i'm wondering if it makes sense to call the prop `isFeedbackSectionHidden` for consistency with other boolean props, and then move the computation into the const above.

```typescript
const isFeedbackSectionHidden = form.responseMode === FormResponseMode.Multirespondent || isFeedbackSubmitted

return (
  //...
  <FormEndPage 
    //...
    isFeedbackSectionHidden={isFeedbackSectionHidden}
    />
  //...
  )

Reply:
77eabe1